docs/source/Tutorial.md

```diff
@@ -1134,6 +1399,14 @@ before:
   }
 ~~~
 </div>
+<div class="language-c">
+~~~{.c}
+  #include "monster_reader.h"
+
+  #define ns(x) FLATBUFFERS_WRAP_NAMESPACE(MyGame_Sample, x) // Specified in the schema.
+  #define nsc(x) FLATBUFFERS_WRAP_NAMESPACE(flatbuffers, x)
```

Comment:
This is a really weird way to do namespaces in C.. I'd recommend doing simple prefixing in the code generator instead. Yes, it makes for longer code, but it is a lot more readable.